pkg/server/datastore/sqlstore/sqlstore_test.go

```diff
@@ -38,8 +37,7 @@ import (
 )
 
 var (
-	ctx       = context.Background()
-	isWindows = runtime.GOOS == "windows"
```

Comment:
This was used below to optionally convert file paths embedded in `file://` URIs on windows. However, the `filepath.ToSlash` method is idempotent if the path already uses forward slashes, so I've updated the code to just pass the paths through filepath.ToSlash unconditionally.